image/templates/helm/stackrox-secured-cluster/internal/defaults/50-images.yaml.htpl

```diff
@@ -24,7 +24,7 @@ image:
   main:
     registry: {{ ._rox.image.registry }}
   collector:
-    registry: {{ if or (eq ._rox.image.registry "stackrox.io") (eq ._rox.image.registry "registry.connect.redhat.com") }}collector.stackrox.io{{ else }}{{ ._rox.image.registry }}{{ end }}
```

Comment:
These are old remnants related to `stackrox.io` -> `collector.stackrox.io` which aren't needed. Cleaning this up since I found it too.

Reply:
Unfortunately, this broke unit tests and requires deeper cleanups. Let me undo the change in this file and defer to the bigger cleanup https://issues.redhat.com/browse/ROX-11642